app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlController.kt

```diff
@@ -660,4 +668,26 @@ class DefaultSessionControlController(
 
         RecentBookmarks.recentBookmarksCount.set(state.recentBookmarks.size.toLong())
     }
+
+    override fun handleMenuItemTapped(item: SearchSelectorMenu.Item) {
+        when (item) {
+            SearchSelectorMenu.Item.SearchSettings -> {
+                navController.nav(
+                    R.id.homeFragment,
+                    HomeFragmentDirections.actionGlobalSearchEngineFragment(),
+                )
+            }
+            is SearchSelectorMenu.Item.SearchEngine -> {
```

Comment:
```suggestion
            SearchSelectorMenu.Item.SearchEngine -> {
```

Reply:
It does look odd, but `SearchEngine` is a data class, so won't work without `is`